rust/worker/src/config.rs

```diff
@@ -570,6 +570,10 @@ mod tests {
             let _ = jail.set_env("CHROMA_COMPACTION_SERVICE__MY_PORT", 50051);
             let _ = jail.set_env("CHROMA_COMPACTION_SERVICE__STORAGE__S3__BUCKET", "buckets!");
             let _ = jail.set_env("CHROMA_COMPACTION_SERVICE__STORAGE__S3__CREDENTIALS", "AWS");
+            let _ = jail.set_env(
```

Comment:
you will need to edit chroma_config.yaml as well I think?

Reply:
As well as https://github.com/chroma-core/k8s/blob/main/chroma-serverless-staging/configuration/staging.yaml

Reply:
yes--I thought this was only in our infra repo, but I guess it's both places

Reply:
7fded1ec